misc/scripts/update_GHpages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Following script copies the markdown files from vmware:master to gh-pages
+
+# get latest changes from master
+git pull
+
+#list files with .md file type
+list_files=$( git ls-files "*.md" )
+
+#Create Temp directory to store .md files
+mkdir ~/backup
```

Comment:
I would suggest to use `/tmp` location to avoid access write/permission issue to write at `~/`